lib/standalone.js

```diff
@@ -81,7 +81,25 @@ module.exports = function (options/*: stylelint$standaloneOptions */)/*: Promise
       code,
       codeFilename: absoluteCodeFilename,
     }).then(postcssResult => {
-      return stylelint._createStylelintResult(postcssResult)
+      // Check for file existence
```

Comment:
It makes a test in https://github.com/stylelint/stylelint/blob/master/lib/__tests__/standalone.test.js#L399-L426 fail otherwise.